src/frame_compare/services/report/assets/viewer.css

```
@@ -330,121 +392,350 @@ button:disabled { opacity: 0.5; cursor: not-allowed; }
+
+.rv-filter-group {
+    display: flex;
+    align-items: center;
+    gap: 0.4rem;
+    overflow-x: auto;
+    width: 100%;
+}
+
+.rv-filter-chip {
+    --category-accent: var(--accent);
+    display: inline-flex;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)